src/app/contacts/main-contacts.component.ts

```diff
@@ -11,6 +12,7 @@ import { ToastrService } from 'ngx-toastr';
 })
 
 export class ContactsMainComponent implements OnInit {
+  @ViewChild(AddContactComponent) addContactComponent: AddContactComponent;
```

Comment:
Super. Zgolj iz radovednosti. A poznaš še kak drug način, kako bi zadevo rešil (brez uporabe @ViewChild)?

Reply:
Sem dodal še rešitev s servisom: Issue #3 with a service.